templates/usr/local/bin/rsync_debian_mirror.j2

```diff
@@ -54,11 +54,22 @@ function writetracefile() {
         fi
     done
 
-    echo "$(date -u)" >> ${base_path}/${trace}.new
-    echo "Used SySync v1" >> ${base_path}/${trace}.new
-    echo "Running on host: ${mirrorname}" >> ${base_path}/${trace}.new
-    echo "Architectures: GUESSED:{${archlist}}" >> ${base_path}/${trace}.new
-    echo "Upstream-mirror: ${repo_source}" >> ${base_path}/${trace}.new
+	# let those tabs be tabs from here...
+	cat <<- EOF >> "$base_path/$trace.new"
+	$(LC_ALL=POSIX LANG=POSIX date -u)
+	Date: $(LC_ALL=POSIX LANG=POSIX date -u -R)
+	Software: SySync v1
+	Running on Host: $mirrorname
+	Architectures: GUESSED:{$archlist}
+	Architecture-Configuration: $globalarchlist
```

Comment:
In your original change, this was `Architectures-Configuration` (note missing `s`). I missed that as well, sorry.